scripts/DCOSWindowsAgentSetup.ps1

```diff
@@ -238,6 +238,16 @@ function New-DockerNATNetwork {
     Write-Output "Created customnat network with flag: com.docker.network.windowsshim.disable_gatewaydns=true"
 }
 
+function Pull-MesosHealthCheckImage{
+    Start-ExecuteWithRetry {
+        docker pull mesos/windows-health-check
```

Comment:
Quick question: We'll pull the `latest` tag over here. Does the `mesos/windows-health-check:latest` point to 1803 ?

Judging by https://hub.docker.com/r/mesos/windows-health-check/tags sizes, the `latest` tag image size seems similar to tag `1709`.